hknweb/templates/base.html

```diff
@@ -151,7 +151,7 @@
       </label>
     </div>
     <div id="about-nav-bar" class="sub-nav-bar navi-container">
-      <a href="">Contact Us</a>
+      <a href="/about">About HKN</a>
```

Comment:
Putting something like `{% url 'about' %}` or something here would be better in case it moves paths (and keeps all the url information in the `urls.py` file. I believe to do this though you'll need to add `name='about'` to the new path added in that file so this knows to find it by that name.

There's also a bunch of other stuff in this file that doesn't use the `url` template helper but probably should, but that's to do in another review instead.